doc/design.md

```diff
@@ -215,6 +215,45 @@ Returns:
     * tbd
 
 
+Data Model
+----------
+
+#### Record:
+ - cases :: type list[Case]
+
+#### Case:
+ - name
+ - birth_year
+ - case_number
+ - citation_number
+ - date
+ - location
+ - violation_type
+ - current_status
+ - balance_due_in_cents
```

Comment:
Did we change this to `balance_due`?

Reply:
I left it as balance_due_in_cents, because calling both that and the getter/setter balance_due seems confusing and bug-prone

Reply:
OK. Thanks